### ISSUE TITLE
statik consume lot of memory

### ISSUE THREAD

**Issue by segator** [rakyll/statik#115]
Can we support writting on disk instead on memory? I trying to embed 1gb of content thats mean use 1gb of memory by default :(

**Comment by sanan-fataliyev** [comment]
>  If your static files are not large in size and will be browsed by a few people, statik is a solution you are looking for.

**Comment by segator** [comment]
Yes I know how to read.., I'm just asking if we can support writting on disk too.

**Comment by pmallela** [comment]
you don't need this library, you can just use os.Open
